packages/contracts/test/core/permission/permission-manager.ts

```diff
@@ -21,15 +21,15 @@ const RESTRICTED_PERMISSIONS_FOR_ANY_ADDR = [
   ethers.utils.id('TEST_PERMISSION_2'),
 ];
 
-const UNSET_FLAG = ethers.utils.getAddress(
+export const UNSET_FLAG = ethers.utils.getAddress(
   '0x0000000000000000000000000000000000000000'
 );
-const ALLOW_FLAG = ethers.utils.getAddress(
+export const ALLOW_FLAG = ethers.utils.getAddress(
   '0x0000000000000000000000000000000000000002'
 );
+export const ANY_ADDR = '0xffffffffffffffffffffffffffffffffffffffff';
```

Comment:
should we think about a global constant for the tests?

Reply:
Let's refactor this when we work on the `osx-commons` part. I will remove it for now again.

Reply:
Reverted in e82dc3ac48ee04c29cb0c875122ed702655baec3